pkg/controller/pravegacluster/upgrade_test.go

```diff
@@ -170,7 +170,7 @@ var _ = Describe("Pravega Cluster", func() {
 
 				It("should set upgrade condition reason to UpgradingBookkeeperReason and message to 0", func() {
 					_, upgradeCondition := foundPravega.Status.GetClusterCondition(pravegav1alpha1.ClusterConditionUpgrading)
-					Ω(upgradeCondition.Reason).Should(Equal(pravegav1alpha1.UpgradingBookkeeperReason))
+					Ω(upgradeCondition.Reason).Should(Equal(pravegav1alpha1.UpdatingBookkeeperReason))
```

Comment:
The only changes to testing are naming changes, can we add test cases to validate functionality?

Reply:
I have added these today. Will add more tomorrow.